.editorconfig

```diff
@@ -0,0 +1,7 @@
+root = true
```

Comment:
This file should be added to gitignore on OS level

Reply:
Why is that? It defines **your** code style, not your OS code style.

Reply:
This library is platform-independent. We don't need this `.editorconfig` file for sure. Please exclude it on your side via global `.gitignore`